ccm_web/server/src/api/api.admin.handler.ts

```diff
@@ -69,16 +72,30 @@ export class AdminApiHandler {
     return parentAccounts
   }
 
+  logTooManyCourseResultsWarning (queryParams: AccountCoursesQueryParams): void {
+    const relevantParams = { by_teachers: queryParams.by_teachers, search_term: queryParams.search_term }
+    logger.warn('Query with the following search term(s) returned too many results: ' + JSON.stringify(relevantParams))
```

Comment:
```suggestion
    logger.warn(`Query with the search term(s) ${JSON.stringify(relevantParams)} returned too many results`)
```

Reply:
Was there something wrong with what I had before?

If I change it to what you have there will be an object in the middle of the sentence which could be confusing looking.